packages/frontend/src/features/common/styles/constans/FontSize.tsx

```diff
@@ -5,4 +5,5 @@ export enum FontSize {
     headerMobile = '30px',
     subtitleDesktop = '22px',
     subtitleMobile = '20px',
+    MEDIUM = "20px"
```

Comment:
tutaj tak samo trzymaj sie jednej formy nazewnictwa i w sumie jak to są enumy to powinny być wszystkie z duzych liter